src/desktop/paths.hpp

```diff
@@ -54,17 +55,18 @@ std::ostream& operator<<(std::ostream& os, const path_info& pinf);
 
 enum GAME_PATH_TYPES
 {
-	GAME_BIN_DIR = 0x1,				/**< Game executable dir. */
-	GAME_CORE_DATA_DIR = 0x2,		/**< Game data dir. */
-	GAME_USER_PREFS_DIR = 0x4,		/**< User preferences dir. */
-	GAME_USER_DATA_DIR = 0x8,		/**< User data dir. */
+	GAME_BIN_DIR = 1,				/**< Game executable dir. */
```

Comment:
It seems more convenient given the options provided are now variable instead of using the hardcoded defaults.

Reply:
I mean which part is more convenient? Calling system_path is now more verbose since the default is gone and calling game_path is just different? Anyway, doesn't matter much. There is just a single caller. And now we're ready to add more than 16 game paths. :smile: